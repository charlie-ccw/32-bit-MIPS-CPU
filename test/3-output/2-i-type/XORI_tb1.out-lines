register_v0_final 000001e7
