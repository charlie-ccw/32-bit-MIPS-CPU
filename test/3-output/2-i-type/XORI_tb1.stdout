RAM : INIT : Loading RAM contents from test/1-binary/2-i-type/xori_tb1.txt
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x2463019c
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x3862007b
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x00000008
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x00000000
TB : INFO : RAM_ACCESS: Read from 0x00000000, data: 0x00000000
register_v0_final 000001e7
test/bus_cpu_tb.v:52: $finish called at 310 (1s)
